visidata/threads.py

```diff
@@ -465,6 +486,7 @@ def allThreadsSheet(self):
     'Progress': Progress,
     'asynccache': asynccache,
     'asyncsingle': asyncsingle,
```

Comment:
I like your new name.  We should think about renaming this to `asyncsingle_cancel`.